contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Groups
+sidebar: Docs
+showTitle: true
+---
+
+# What can you do with groups?
+
+Groups are how you track and perform analysis on the aggregation of an something other than a user. 
+
+## Group Analytics
+You can perform any type of analytics (e.g. Trends, Funnels, Retention, etc.) based on groups rather than users (e.g. Number of companies who signed up in the last 28 days, Number of total transactions in the past 14 days, etc.)
+
+## Feature flags
+You can use groups to roll-out your feature flags more consistently. Instead of giving users in the same organization a different experience, roll out the feature flag to whole organizations.
+
+# How our solution works?
+
+With groups, there's two important concepts to remember. These are Groups and Group Types.
+
+## Groups vs Group Types
+
+The simplest example of a group type is a company, users from multiple companies that might use your product - groups allows you do to analytics on these - e.g. Daily Active Companies.
```

Comment:
Should be "groups allows you **to do** analytics"

Reply:
```suggestion
The simplest example of a group type is a company; users from multiple companies that use your product. Groups enable you to analyze these - e.g. Daily Active Companies.
```